### PR TITLE
chore adding extra details to unit test scenarios

### PR DESCRIPTION
**Pull request by nicky6s**
I have a future change in mind to do with CTEs. When I make the PR for this change I would like the unit tests changes (if any) to be simple and easy to understand. So, before I start to look at my CTE related change, I've added extra details to the `UpdateTest`, `InsertTest`, `DeleteTest` and `SelectTest` unit test scenarios.

There are zero functionality changes made here, only unit tests updates, I've been very careful not to change any SQL statements being tested.

### CODE REVIEW

File: src/test/java/net/sf/jsqlparser/statement/delete/DeleteTest.java [hunk 1]
```diff
@@ -118,8 +123,19 @@ public void testWith() throws JSQLParserException {
                 + "DELETE FROM cfe.instrument_ref\n"
                 + "WHERE  id_instrument_ref = (SELECT id_instrument_ref\n"
                 + "                            FROM   a)";
-
         assertSqlCanBeParsedAndDeparsed(statement, true);
```
Comment (manticore-projects) [inline src/test/java/net/sf/jsqlparser/statement/delete/DeleteTest.java:126]: Please note that `assertSqlCanBeParsedAndDeparsed` returns the parsed `Statement` and so you can simplify to

```java
Delete delete = (Delete)  assertSqlCanBeParsedAndDeparsed( sqlStr, true );
```

which saves you one parse step.

Please apply to all your changed tests.

**Review by manticore-projects — CHANGES_REQUESTED** [review]
Please refactor the duplicate/redundant parser calls.

File: src/test/java/net/sf/jsqlparser/statement/delete/DeleteTest.java (hunk 1 above)
Comment (nicky6s) [reply src/test/java/net/sf/jsqlparser/statement/delete/DeleteTest.java]: Thanks @manticore-projects - Ive just pushed [this commit](https://github.com/JSQLParser/JSqlParser/pull/2051/commits/0566fb0883fd923ef571e9ce4e132b0f38648aa1) to address this

**Comment by manticore-projects** [comment]
Thank you for this contribution.

**Pull request merged**